planning/behavior_path_planner/src/utils/geometric_parallel_parking/geometric_parallel_parking.cpp

```diff
@@ -377,18 +381,21 @@ std::vector<PathWithLaneId> GeometricParallelParking::planOneTrial(
   const double goal_yaw = tf2::getYaw(arc_end_pose.orientation);
   const double psi = normalizeRadian(self_yaw - goal_yaw);
 
-  const Pose Cr = calcOffsetPose(arc_end_pose, 0, -R_E_r, 0);
-  const double d_Cr_Einit = calcDistance2d(Cr, start_pose);
+  const Pose Cfar = left_side_parking ? calcOffsetPose(arc_end_pose, 0, -R_E_far, 0)
```

Comment:
C_far seems better same to R_E_far

Reply:
![image](https://github.com/autowarefoundation/autoware.universe/assets/39142679/93eea03f-3cca-49ca-b1a7-b5497757b49f)

Reply:
2e45e98